### DISCUSSION TITLE
How to choose the baudrate ?

### DISCUSSION THREAD

**Discussion by jmcollin78** [darkxst/silabs-firmware-builder#62] [Q&A] [answered]
I buy a Sonoff zdongle-E for being able to do create a Matter/Thread network (in parallel of a Zigbee network with another Skyconnect USB key).

So I want to flash the Zdongle-e ot-rcp-v2.4.1.0-zbdonglee-xxxx. I see there is many (I guess baudrate) xxxx possible. How to take the right one ? Is there any recommandation for this ?

Note: I suppose I have to choose the last v2.4.1 firmware release.

**Comment by darkxst** [comment, marked as answer]
Use 460800 for Thread (or MultiPan)

Web flasher always installs recommended firmware and baudrate. More details on this page:
https://darkxst.github.io/silabs-firmware-builder/
